pandas/io/excel/_base.py

```diff
@@ -1472,19 +1478,28 @@ class ExcelFile:
            - Otherwise if ``path_or_buffer`` is in xlsb format,
              `pyxlsb <https://pypi.org/project/pyxlsb/>`_ will be used.
 
-           .. versionadded:: 1.3.0
+        .. versionadded:: 1.3.0
 
            - Otherwise if `openpyxl <https://pypi.org/project/openpyxl/>`_ is installed,
              then ``openpyxl`` will be used.
            - Otherwise if ``xlrd >= 2.0`` is installed, a ``ValueError`` will be raised.
 
-           .. warning::
+        .. warning::
 
-            Please do not report issues when using ``xlrd`` to read ``.xlsx`` files.
-            This is not supported, switch to using ``openpyxl`` instead.
+           Please do not report issues when using ``xlrd`` to read ``.xlsx`` files.
+           This is not supported, switch to using ``openpyxl`` instead.
+    storage_options : StorageOptions
+        Passed to _get_filepath_or_buffer.
```

Comment:
Can you copy the text that exists for `_shared_docs["storage_options"]`

Reply:
Fixed in L1443 and L1492. Please review.

Reply:
@mroeschke Let me know if I need to improve any other stuff.